sdk/storage/Azure.Storage.DataMovement/tests/PauseResumeTransferTests.cs

```diff
@@ -463,7 +464,11 @@ public async Task PauseThenResumeTransferAsync(TransferDirection transferType)
             await testEventsRaised.AssertPausedCheck();
 
             // Act - Resume Job
-            DataTransferOptions resumeOptions = new DataTransferOptions();
+            DataTransferOptions resumeOptions = new DataTransferOptions()
+            {
+                // Enable overwrite on resume, to overwrite destination.
+                CreationPreference = StorageResourceCreationPreference.OverwriteIfExists
+            };
```

Comment:
Curious why you needed to add this? I tried to keep this out of these tests so we could test that we could get a "clean" pause/resume.

Reply:
We talked about this offline but looks like there's a bug with getting a clean pause and updated checkpointer file. I added a bug for that to address in a separate PR.